spec/e2e/setup.ts

```diff
@@ -0,0 +1,8 @@
+import fs from "node:fs/promises";
```

Comment:
Again, not sure if it matters (or if it's a config error on my end), but this is giving me a "no default export" warning, so use this if appropriate:
```suggestion
import * as fs from "node:fs/promises";
```
This follows the ESM-style imports shown in [Node docs](https://nodejs.org/api/module.html).

Though it must not matter after all, since the tests passed in CI, so feel free to ignore this.

Reply:
Yeah, this apparently works. Weird.